spec/rails_app/config/application.rb

```diff
@@ -1,10 +1,8 @@
 require File.expand_path('../boot', __FILE__)
 
 require "active_model/railtie"
-require "active_job/railtie"
```

Comment:
Pre-emptive comment: I had trouble even using the gem because of those lines here. I figured they weren't really needed so I removed them. I decided to keep it unless you want it back, so I'm gonna make another commit to roll this back.